src/store/metadata/stub/content/editions/versions/additions-to-the-housing-stock_2022-2023.json

```diff
@@ -0,0 +1,125 @@
+                "name": "Period",
+                "datatype": "string",
+                "titles": "Period",
+                "description": "The time period associated with the observation."
+              },
+              {
+                "name": "AREACD",
+                "datatype": "string",
+                "titles": "Statistical Geography",
+                "description": "The geography code associated with the observation."
+              },
+              {
+                "name": "AREANM",
+                "datatype": "string",
+                "titles": "Area Name",
+                "description": "The name assiciated with The geography code "
+              },
+              {
+                "name": "Geography",
+                "datatype": "string",
+                "titles": "Geography Type",
+                "description": "The categorisation of the geography code"
+              },
+              {
+                "name": "Variable Name",
+                "datatype": "string",
+                "titles": "Variable Name",
+                "description": "Which ESS variable is being measured"
+              },
+              {
+                "name": "Indicator", 		
+                "datatype": "string",
+                "titles": "Indicator",
+                "description": "Which ESS indicator is being measured"
+              },
+              {
+                "name": "Mission",
+                "datatype": "string",
+                "titles": "Mission",
+                "description": "Which ESS mission number this dataset is fulfilling"
+              },
+              {
+                "name": "Category",
+                "datatype": "string",
+                "titles": "Category",
+                "description": "Which ESS category the dataset belongs to"
+              },
+              {
+                  "name": "Observation",
+                  "datatype": "integer",
+                  "titles": "Observation",
+                  "description": "Observation value for the chosen variable"
+                },
+                {
+                    "name": "Measure",
+                    "datatype": "string",
+                    "titles": "Measure Type",
+                    "description": "Measure Type of the Observational Value"
+                },
+                {
+                    "name": "Unit",
+                    "datatype": "string",
+                    "titles": "Unit",
+                    "description": "Unit Type of the Observational Value"
+                  },
+                {
+                "name": "Lower Confidence Interval (95%)",
+                "datatype": "decimal",
+                "titles": "Lower Confidence Interval",
+                "description": "The interval estimate gives an indication of how much uncertainty there is in our estimate of the true mean, Lower bound"
+              },
+              {
+                "name": "Upper Confidence Interval (95%)",
```

Comment:
should be snake_case, like in `apprenticeships-achievements_2023-05.json`
`
"name": "upper_confidence_interval_95",`